source/_integrations/azure_storage.markdown

```
@@ -0,0 +1,46 @@
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
+
+{% include integrations/remove_device_service.md %}
+
+## Troubleshooting
+
+{% details "Authentication failure" %}
+
+Check that your storage account allows (`Shared Key` access)[https://learn.microsoft.com/en-us/azure/storage/common/shared-key-authorization-prevent?tabs=portal#remediate-authorization-via-shared-key].
```

Markdownlint (0.37.0)

(`Shared Key` access)[https://learn.microsoft.com/en-us/azure/storage/common/shared-key-authorization-prevent?tabs=portal#remediate-authorization-via-shared-key]
Reversed link syntax

(MD011, no-reversed-links)